WebContent/WOH-addRestaurant.jsp

```diff
@@ -40,6 +39,70 @@
     	</ul>
   	</div>
 	</nav>
+	
+	<div class ="container">
+	<div class="row">
+			<div class="col-sm-10 col-sm-offset-1">
+			<form action = "" method = "POST" class="form-horizontal" id="">
+					
+				
+				
+				<!-- Restaurant Name Textbox -->
+					<div class="form-group">
+					<label class="control-label col-sm-2" for="item">Restaurant Name</label>
+						<div class="col-sm-10">
+         					<input type = "text" class="form-control" name = "restaurantName">
+         				</div>
+         			</div>
+         			
+				<!-- Average Price Textbox -->
+					<div class="form-group">
+					<label class="control-label col-sm-2" for="averagePrice">Price Per Meal</label>
+						<div class="col-sm-10">
+         					<input type = "text" class="form-control" name = "averagePrice">
+         				</div>
+         			</div>
+				
+				<!-- Distance Textbox(If we use this) -->
+					<div class="form-group">
+					<label class="control-label col-sm-2" for="distance">Distance</label>
+						<div class="col-sm-10">
+         					<input type = "text" class="form-control" name = "distance">
+         				</div>
+         			</div>
+         			
+         		<!-- Food Type Dropdown -->
+					<div class="form-group">
+  					<label class="control-label col-sm-2" for="foodType">Select Food Type</label>
+  						<div class="col-sm-10">
+  							<select class="form-control" name="foodType">
+    							<option value = "Fast-Food" >Fast Food</option>
+    							<option value = "Sit-Down" >Sit-Down</option>
+    							<option value = "Drive-Through" >Drive-Through</option>
+  							</select>
+  						</div>
+  					</div>
+  					
+  					<!-- Cuisine Type Checkbox -->
+  					<div class="form-group"> 
```

Comment:
I would recommend setting a specific height to this div and adding "overflow-y: scroll".  This will allow us to add a whole lot more cuisine options in a neat little package.